src/utils/tokens.ts

```diff
@@ -361,6 +364,28 @@ const tokens = {
       });
     }
 
+    if (data !== null) {
+      for (const dataString of data) {
+        const outputData = {
+          type: 'data',
+          data: dataString,
+          value: 1,
+          token: HATHOR_TOKEN_CONFIG.uid,
+          authorities: 0,
+        } as IDataOutput;
+
+        // We currently have an external service that identifies NFT tokens with the first output as the data output
+        // that's why we are keeping like this
+        // However, this will change after a new project is completed to better identify an NFT token
+        // the method that validates the NFT is in src/models/CreateTokenTransaction.validateNft
```

Comment:
I don't get it, we already have the `validateNft` method. What is missing?

If something is really missing, can you improve this comment to be clear on what it is?

Reply:
This comment was already present on the `prepareCreateTokenData` where we were adding the data outputs.
I just moved the code to the `prepareMintTxData` because we now want this behavior on both methods.